src/interpreter.cpp

```diff
@@ -626,7 +626,7 @@ struct command_info cmd_info[] =
     { "domain"     , POS_LYING   , do_domain   , 0, 0, BLOCKS_IDLE_REWARD },
     { "download"   , POS_RESTING , do_download_headware, 0, 0, BLOCKS_IDLE_REWARD },
     { "donate"     , POS_RESTING , do_drop     , 0, SCMD_DONATE, BLOCKS_IDLE_REWARD },
-    { "drag"       , POS_STANDING, do_drag     , 0, 0, BLOCKS_IDLE_REWARD },
+    { "drag"       , POS_SITTING , do_drag     , 0, 0, BLOCKS_IDLE_REWARD },
```

Comment:
Just to confirm, this worked when you tested it? Making a command suddenly work in vehicles is a leading cause of crashes due to unguarded references of ch->in_room.

Reply:
Ah, a bit more complicated than I thought. Might actually be better to put the functionality into do_pull instead. Anyway, reverted this bit.